src/doc/footer.inc

```diff
@@ -5,5 +5,5 @@ or the <a href="http://opensource.org/licenses/MIT">MIT license</a>, at your opt
 </p><p>
 This file may not be copied, modified, or distributed except according to those terms.
 </p></footer>
-<script type="text/javascript" src="jquery.js"></script>
-<script type="text/javascript" src="playpen.js"></script>
+<script type="text/javascript" src="/jquery.js"></script>
+<script type="text/javascript" src="/playpen.js"></script>
```

Comment:
Do you have any suggestions on a fix that works locally and at multiple levels in the tree? Not super familiar here.